pkg/render/monitor.go

```diff
@@ -268,3 +273,58 @@ func (mc *monitorComponent) serviceMonitorElasicsearch() *monitoringv1.ServiceMo
 		},
 	}
 }
+
+func (mc *monitorComponent) role() *rbacv1.Role {
+	// Even though list and watch verbs are listed here, they are still cluster scoped
```

Comment:
Would this comment be better where the actual Cluster scoped Role is?  (I assume that still exists somewhere else?)

Reply:
I see - they're in the other repo.  I guess it was annoying to change kube-controllers to be more specific in its watch?